src/js/components/tasks/createForm.tsx

```diff
@@ -64,10 +64,10 @@ const CreateTaskModal = ({
   };
 
   const additionalData: { [key: string]: any } = {
-    epic: epic.id,
+    epic: epic?.id,
```

Comment:
If we don't have an `epic`, then we need to pass in a `project` here:

```suggestion
    epic: epic?.id,
    project: epic ? undefined : project.id,
```